interface/attribute-editor.md

```diff
@@ -1,14 +1,11 @@
 # Attribute Editor
 
-**Attributes** are features that are applied to **Objects** in Incari, so that the **Objects** can exhibit specific behaviors in a **Scene**.
+**Attributes** are features that are applied to **Objects** in Incari, so that the **Objects** can exhibit specific properties and behaviors in a **Scene**.
 
 The **Attribute Editor** holds the **Attributes** for all **Objects** in Incari and therefore, **Attributes** are edited here.
 
-While some **Attributes** are shared between **Objects**, others are unique in which **Objects** they belong to. In order to access the **Attributes** of an **Object**, The **Attribute Editor** is opened and the **Object** is selected in the **Scene Outliner** by **clicking** on it. This will show the **Attributes** in the **Attribute Editor**. 
-
-If the **Attribute Editor** is not opened, it can be opened by *right-clicking* on the *Menu bar* of Incari Studio and subsequently clicking on **Attribute Editor** from the drop-down list. This is seen in the illustration below:
-
-![](../.gitbook/assets/interface/attribute-editor/attribute-editor.PNG)
-
+While some **Attributes** are shared between **Objects**, others are unique in which **Objects** they belong to and in order to access the **Attributes** of an **Object**, the **Object** is selected in the **Scene Outliner** by **clicking** on it. This will show the **Attributes** in the **Attribute Editor**. 
```

Comment:
I just rephrased this a little because the "**Attributes** are shared between **Objects**" line could be perceived as meaning that changing an attribute on one object also affects a different object.